### PR TITLE
Dashing ov2.0

### PR DESCRIPTION
**Pull request by ChloeWu1**
Update API for ov 2.0

### CODE REVIEW

File: .ci_local_test/Jenkinsfile [hunk 1]
```diff
@@ -30,6 +31,7 @@ pipeline {
             steps {
                 script {
                     def test_result = sh script: "ssh intel@$Test_Server 'cd $Test_WORKSPACE && ./self_container_ros2_openvino_dashing_test.sh '", returnStatus: true
+>>>>>>> origin/dashing
```
Comment (huangjiafengx) [inline .ci_local_test/Jenkinsfile:34]: The line ">>>>>>> origin/dashing" need remove.

File: doc/quick_start/getting_started_with_Dashing_Ubuntu18.04_ov2.0.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  Tips: Both runtime and development tool are needed, `pip` is recommended for installing the development tool ([guide](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html))
+  * Install from source code ([guide](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux))
+* Install Intel® RealSense™ SDK ([guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md))
+
+## 2. Building and Installation
+* Install ROS2_OpenVINO packages
+```
+mkdir -p ~/catkin_ws/src
+cd ~/catkin_ws/src
+git clone https://github.com/intel/ros2_openvino_toolkit -b dashing
```
Comment (huangjiafengx) [inline doc/quick_start/getting_started_with_Dashing_Ubuntu18.04_ov2.0.md:19]: image
there have the 5 numbers to git clone, but 4 numbers git clone in the dockerfile

Comment (ChloeWu1) [reply doc/quick_start/getting_started_with_Dashing_Ubuntu18.04_ov2.0.md:19]: Some package could be removed.

**Comment by LewisLiuPub** [comment]
LGTM.
Branch **dashing** (for ov2.0) & **dashing-ov_legacy** (ov2021.4LTS) will be deprecated. only basic function tests have been done for the upgrading in Y2022.

**Pull request merged**